### ISSUE TITLE
About accuracy of the pretrained vgg.npy

### ISSUE THREAD

**Issue by hjh3169** [machrisaa/tensorflow-vgg#67]
I downloaded the pretrained model vgg16.py and tested it on the ImageNet validation set. However I found that the accuracy is not as high as that given on Caffe. I only get 66.9% of Top-1 accuracy...
